lib/zone.ts

```diff
@@ -1338,7 +1343,19 @@ const Zone: ZoneType = (function(global: any) {
     return '__zone_symbol__' + name;
   }
 
-
   performanceMeasure('Zone', 'Zone');
+  const z: any = Zone.prototype;
+  z.w = z.wrap;
+  z.f = z.fork;
+  z.r = z.run;
+  z.rg = z.runGuarded;
+  z.rt = z.runTask;
+  z.st = z.scheduleTask;
+  z.sc = z.scheduleMacroTask;
+  z.si = z.scheduleMicroTask;
+  z.se = z.scheduleEventTask;
+  z.ct = z.cancelTask;
+  (Zone as any).l = Zone.__load_patch;
```

Comment:
same thing here, helpers in `util.ts` can hide the long name without loosing readability

Reply:
Got it.